.github/workflows/test.yml

```diff
@@ -213,6 +213,34 @@ jobs:
 
       - name: Integration Tests
         run: pnpm -- turbo run integration-tests --filter=cli
+
+  go_e2e:
+    name: Go E2E Tests
+    needs: determine_jobs
+    if: needs.determine_jobs.outputs.go == 'true'
```

Comment:
We should also run if anything in `crates/turborepo*` changes along with global changes like `Cargo.lock`

Reply:
updated and also included prysk tests for this condition if you want to take a look